help/move-to-zulip-cloud.md

```diff
@@ -0,0 +1,100 @@
+# Move to Zulip Cloud from self-hosting
+
+With Zulip's high quality import and export tools, you can always
+move from self-hosting your own Zulip server to using our Zulip
+Cloud service (and back).
+
+## Advantages of Zulip Cloud
+
+{!advantages-of-zulip-cloud.md!}
+
+## Process overview
+
+To move your Zulip organization from your self-hosted server to Zulip
+Cloud, you will need to take the following steps, which are described
+in more detail below:
+
+{start_tabs}
+
+1. [Coordinate with Zulip Cloud support](#coordinate-with-zulip-cloud-support).
+
+2. [Upgrade server to `zulip-cloud-current` branch](#upgrade-server).
```

Comment:
Probably this should mention "Upgrade your self-hosted server ..." for clarity.

Reply:
Done.